site/app/templates/GlobalHeader.twig

```diff
@@ -223,6 +223,10 @@
                 {{ system_message }}
             </div>
             {% endif %}
+            <div class="system-message warning" id="socket-server-system-message" hidden>
+                Warning: Failed to connect to websocket server. This page will not dynamically update.
+                Manually reload this page to see updates. Notify the sysadmin {{ sysadmin_email }}.
```

Comment:
We should check that `sysadmin_email` is non-empty before showing the "Notify the sysadmin" bit. Additionally, I would say change the final sentence te:

```
Notify the sysadmin at <a href='mailto:{{ sysadmin_email }}'>{{ sysadmin_email }}</a>.
```

To make use of the mailto tag, minimally for screen readers if not others.